src/network/chain.py

```diff
@@ -24,3 +29,9 @@ def __init__(
         self.credentials = credentials
         self.tracked_contracts = tracked_contracts
         self.active = active
+        if config.VALIDATE_CHAINS:
```

Comment:
If we intend on moving the validation back to `get_connection` then would it make sense to add this as an argument?  The signature would then be `get_connection(self, validate_chain: bool)`.

Reply:
Not necessarily essential, but it could not hurt if we default it to `true`.

Reply:
I've set the default to `true` as advised.  I think we need to consider how we want to specify whether a chain should be validated.  Two options spring to mind.  We can use an environment variable or we can add an argument to each chains configuration.  Going the chain configuration route gives more granular control.  It may be sufficient to leave with the default set to `true` for the minute and see how the requirements for this feature play out in practice.